Mathlib/Data/List/DropRight.lean

```diff
@@ -241,10 +241,30 @@ theorem mem_rtakeWhile_imp {x : α} (hx : x ∈ rtakeWhile p l) : p x := by
   exact mem_takeWhile_imp hx
 #align list.mem_rtake_while_imp List.mem_rtakeWhile_imp
 
-variable (p) (l)
-
-theorem rtakeWhile_idempotent : rtakeWhile p (rtakeWhile p l) = rtakeWhile p l :=
+theorem rtakeWhile_idempotent (p : α → Bool) (l : List α) :
```

Comment:
This is a drive-by clean-up: my understanding is this is worthwhile anyway. It also makes the next lines nicer to write, as I can re-use `l`.